### PR TITLE
xcodes, xcodes-runtimes: add page

### PR DESCRIPTION
**Pull request by marc-48k**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** 1.4.1

### CODE REVIEW

File: pages/osx/xcodes-runtimes.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+`xcodes runtimes --include-betas`
+
+- Download a Simulator runtime:
+
+`xcodes runtimes download "{{runtime-name}}"`
+
+- Download and install a Simulator runtime:
+
+`xcodes runtimes install "{{runtime-name}}"`
```
Comment (acuteenvy) [inline pages/osx/xcodes-runtimes.md:16]:
```suggestion
- Download a Simulator runtime:

`xcodes runtimes download {{runtime-name}}`

- Download and install a Simulator runtime:

`xcodes runtimes install {{runtime-name}}`
```

File: pages/osx/xcodes.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+`xcodes installed`
+
+- List all available Xcode versions:
+
+`xcodes list`
+
+- Select an Xcode version by specifying a version number or path:
+
+`xcodes select {{xcode-version|/path/to/Xcode.app}}`
```
Comment (acuteenvy) [inline pages/osx/xcodes.md:16]:
```suggestion
- Select an Xcode version by specifying a version number or a path:

`xcodes select {{xcode-version|path/to/Xcode.app}}`
```

File: pages/osx/xcodes.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+- Download and install a specific Xcode version:
+
+`xcodes install {{xcode-version}}`
+
+- Install the latest 'release' Xcode version and select it:
```
Comment (acuteenvy) [inline pages/osx/xcodes.md:22]:
```suggestion
- Install the latest Xcode release and select it:
```

File: pages/osx/xcodes.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+
+`xcodes install --latest --select`
+
+- Download a specific Xcode version archive to a given directory without installing it:
+
+`xcodes download {{xcode-version}} --directory {{path/to/download/directory}}`
```
Comment (acuteenvy) [inline pages/osx/xcodes.md:28]:
```suggestion
`xcodes download {{xcode-version}} --directory {{path/to/directory}}`
```

File: pages/osx/xcodes.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
+# xcodes
+
+> Download, install and manage multiple Xcode versions.
+> More information: <https://github.com/xcodesorg/xcodes>.
```
Comment (acuteenvy) [inline pages/osx/xcodes.md:4]:
```suggestion
> Download, install and manage multiple Xcode versions.
> See also: `xcodes runtimes`.
> More information: <https://github.com/xcodesorg/xcodes>.
```

**Review by acuteenvy — APPROVED** [review]

**Review by pixelcmtd — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution and welcome to tldr.

**Pull request merged**